src/main/default/lwc/lookup/lookup.js

```diff
@@ -63,6 +71,21 @@ export default class Lookup extends LightningElement {
             }
             return result;
         });
+        const self = this;
```

Comment:
After this line I would also include:
this.focusedResultIndex = null;

To make sure index is cleared on new search results.